Cargo.toml

```diff
@@ -25,6 +25,7 @@ log = "~0.3.3"
 crossbeam = "~0.1.6"
 memmap = "0.2.2"
 maidsafe_utilities = "~0.1.3"
+ip = "1.0.0"
```

Comment:
Shouldn't we use `ip = "~1.0.0"`?  (same for memmap).  I guess that can be updated in the next PR to avoid blocking this one from being merged.